src/utils/KML.js

```diff
@@ -382,16 +402,23 @@ const writeFeatures = (layer, featureProjection) => {
         clone.set('iconRotation', newStyle.image.getRotation());
       }
 
-      // Set zoomAtMaxIconSize to use for icon-to-map proportional scaling
-      if (f.get('zoomAtMaxIconSize')) {
-        clone.set('zoomAtMaxIconSize', f.get('zoomAtMaxIconSize'));
-        newStyle.fill = null;
+      /* Replaced by mapResolutionForPicture, to be removed in react-spatial v2 */
```

Comment:
I guess you can remove it plan-editor will take care of the backward compataibility

Reply:
Yes